site/content/docs/user/quick-start.md

```diff
@@ -367,14 +357,9 @@ Ensure you `systemctl restart docker` to pick up the changes.
 ```yaml
 # an ipv6 cluster
 kind: Cluster
-apiVersion: kind.sigs.k8s.io/v1alpha3
+apiVersion: kind.sigs.k8s.io/v1alpha4
```

Comment:
kind.x-k8s.io/v1alpha4 here and everywhere else

Reply:
done